lib_stusb_impl/usbd_impl.c

```diff
@@ -927,6 +944,30 @@ uint8_t *USBD_HID_GetReportDescriptor_impl(uint16_t *len)
     return 0;
 }
 
+#ifdef HAVE_USB_CLASS_CCID
+/**
+ * @brief  Returns the pinpad value offset in the descriptor.
+ * @retval Offset
+ */
+volatile uint8_t *USBD_GetPinPadOffset(void)
+{
+    unsigned short length  = 0;
+    uint8_t       *cfgDesc = NULL;
+    unsigned short offset  = 0;
+
+    cfgDesc = USBD_GetCfgDesc_impl(&length);
+
+    offset = CFG_HDR_LEN + CFG_HIDGEN_LEN;
+#ifdef HAVE_IO_U2F
+    offset += CFG_IO_U2F_LEN;
+#endif  // HAVE_IO_U2F
+    offset += 61;
+
+    // Returns a volatile pointer allowing callers to do read - write operations
+    return (volatile uint8_t *) (cfgDesc + offset);

```

Comment:
Proposition:
```suggestion
    // Returns a const volatile pointer allowing callers to do/
    // - write operations through nvram_write
    // - read operation without potential compilation optimization issue thanks to the volatile qualifier.
    return (const volatile uint8_t *) (cfgDesc + offset);
```
https://stackoverflow.com/a/4597164

Reply:
done